### PR TITLE
[Pre-Training] Enhance ERNIE pretrain with additional chinese char

### PR DESCRIPTION
**Pull request by ZHUI**
### PR types

 Function optimization
### PR changes

Others
### Description

Enhance ERNIE pretrain with additional chinese char

### CODE REVIEW

File: paddlenlp/transformers/ernie/tokenizer.py [hunk 1]
```diff
@@ -236,6 +236,48 @@ def vocab_size(self):
...
+        For, char level model such as ERNIE, we need add ## chinese token 
+        to demonstrate the segment information.
+        """
+        vocab_set = set(self.vocab.token_to_idx.keys())
+        extend_list = []
+        for i in range(len(self.vocab)):
+            if i not in self.vocab.idx_to_token:
+                continue
+            w = self.vocab.idx_to_token[i]
+            if len(w) == 1 and ord(w) >= 0x4E00 and ord(w) <= 0x9FA5:
```
Comment (ZeyuChen) [inline paddlenlp/transformers/ernie/tokenizer.py:367]: 补充注释

Comment (ZHUI) [reply paddlenlp/transformers/ernie/tokenizer.py:367]: done

File: model_zoo/ernie-1.0/data_tools/dataset_utils.py [hunk 2]
```diff
@@ -83,8 +83,9 @@ def __init__(self, datasets, weights):
         self.dataset_index = np.zeros(self.size, dtype=np.uint8)
         self.dataset_sample_index = np.zeros(self.size, dtype=np.int64)
 
-        local_rank = 0 if fleet.local_rank() is None else int(
-            fleet.local_rank())
+        # local_rank = 0 if fleet.local_rank() is None else int(fleet.local_rank(
```
Comment (LemonNoel) [inline model_zoo/ernie-1.0/data_tools/dataset_utils.py:86]: 这里注释可以删掉吗

File: model_zoo/ernie-1.0/run_pretrain_static.py [hunk 3]
```diff
@@ -187,12 +205,15 @@ def dist_optimizer(args, topo):
     exec_strategy.num_iteration_per_drop_scope = 10000
 
     build_strategy = paddle.static.BuildStrategy()
-    #build_strategy.enable_sequential_execution = True # for profile
+    build_strategy.enable_sequential_execution = True  # for profile
```
Comment (LemonNoel) [inline model_zoo/ernie-1.0/run_pretrain_static.py:208]: profile 相关的代码注释或者删除掉？

**Review by LemonNoel — APPROVED** [review]
LGTM

**Pull request merged**